src/store/wallet.js

```diff
@@ -531,8 +545,9 @@ export default {
       if (request.result) {
         context.commit('stopSessionTimeout')
         context.commit('deleteSession')
-        context.state.unlockedWalletDescription = null
+        context.state.description = null
```

Comment:
This should be done in a mutation

Reply:
are you commenting your own PR? :rofl:

Reply:
I find it really useful to track last touches or improvements I can make...  :flushed: